src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -244,6 +246,19 @@ internal static unsafe string AnsiBstrBufferToString(byte* buffer)
             return Marshal.PtrToStringAnsi((IntPtr)buffer, (int)Marshal.SysStringByteLen((IntPtr)buffer));
         }
 
+#if FEATURE_OBJCMARSHAL
```

Comment:
Shouldn't this go to `InteropHelpers.Apple.cs` instead of adding more ifdefs?

Reply:
Done.